.github/workflows/container-tests.yml

```diff
@@ -4,112 +4,25 @@ on:
-            tf_scope: "private"
-          - tmt_plan: "c9s"
-            os_test: "c9s"
-            context: "CentOS Stream 9"
-            compose: "CentOS-Stream-9"
-            api_key: "TF_PUBLIC_API_KEY"
-            branch: "main"
-            tmt_repo: "https://github.com/sclorg/sclorg-testing-farm"
+        os_test: [ "fedora", "centos7", "rhel7", "rhel8", "rhel9", "c9s", "rhel9-unsubscribed" ]
+        test_case: [ "container" ]
 
     if: |
       github.event.issue.pull_request
       && (contains(github.event.comment.body, '[test]') || contains(github.event.comment.body, '[test-all]'))
       && contains(fromJson('["OWNER", "MEMBER"]'), github.event.comment.author_association)
     steps:
-      - name: Checkout repo
-        uses: actions/checkout@v2
+      - name: Schedule tests for ${{ matrix.version }} - ${{ matrix.os_test }}
+        uses: sclorg/tfaga-wrapper@main
         with:
-          ref: "refs/pull/${{ github.event.issue.number }}/head"
-
-      - name: Prepare needed variables
-        shell: bash
-        id: vars
-        run: |
-          dockerfile="Dockerfile.${{ matrix.os_test }}"
-          if [[ ${{ matrix.os_test }} == "centos7" ]]; then
-            dockerfile="Dockerfile"
-          fi
-          echo "::set-output name=DOCKERFILE_NAME::${dockerfile}"
-
-      - name: Check ${{ matrix.version }}/${{ steps.vars.outputs.DOCKERFILE_NAME }} is present
-        id: check_dockerfile
-        uses: andstor/file-existence-action@v1
-        with:
-          files: "${{ matrix.version }}/${{ steps.vars.outputs.DOCKERFILE_NAME }}"
-
-      - name: Check .exclude-${{ matrix.os_test }} file presents
-        id: check_exclude_file
-        uses: andstor/file-existence-action@v1
-        with:
-          files: "${{ matrix.version }}/.exclude-${{ matrix.os_test }}"
-
-      # https://github.com/sclorg/testing-farm-as-github-action
-      - name: Schedule tests for ${{ matrix.version }} - ${{ matrix.context }}
-        id: github_action
-        if: ${{ steps.check_exclude_file.outputs.files_exists == 'false' && steps.check_dockerfile.outputs.files_exists == 'true' }}
-        uses: sclorg/testing-farm-as-github-action@v1
-        with:
-          api_key: ${{ secrets[matrix.api_key] }}
-          git_url: ${{ matrix.tmt_repo }}
-          git_ref: ${{ matrix.branch }}
-          tf_scope: ${{ matrix.tf_scope }}
-          tmt_plan_regex: ${{ matrix.tmt_plan }}
-          pull_request_status_name: "${{ matrix.context }} - ${{ matrix.version }}"
-          variables: "REPO_URL=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY;REPO_NAME=$GITHUB_REPOSITORY;PR_NUMBER=${{ github.event.issue.number }};SINGLE_VERSION=${{ matrix.version }};OS=${{ matrix.os_test }};TEST_NAME=test"
-          compose: ${{ matrix.compose }}
+          os_test: ${{ matrix.os_test }}
+          test_case: ${{ matrix.test_case }}
+          version: ${{ matrix.version }}
+          private_api_key: ${{ secrets.TF_INTERNAL_API_KEY }}
```

Comment:
This is not needed. It can be done in `tfgag-wrapper` based on the OS host.
Secrets are available over all https://github.com/sclorg organization.